packages/foreigntypes/posixtypes.pony

```diff
@@ -0,0 +1,95 @@
+// This file defines the standard Posix type to make FFI calls easier
+// to manage and more consistent.
+
+type BlkCnt is I64
+
+// not consistent across OSX (always 32 bit int)/Linux (Long)
+type BlkSize is ILong
+type OSXBlkSize is I32
+
+type Cc is U8
+
+// not consistent across OSX (Unsigned Long)/Linux (Long)
+type Clock is ILong
+type OSXClock is ULong
+
+// not defined on OSX
+type ClockId is I32
+
+// not consistent across OSX (always 32 bit int)/Linux (always 64 bit)
+type Dev is I64
+type OSXDev is I32
+
+// not consistent across OSX (always unsigned int)/Linux (always unsigned long)
+type FsBlkCnt is ULong
+type OSXFsBlkCnt is U32
+
+// not consistent across OSX (always unsigned int)/Linux (always unsigned long)
+type FsFilCnt is ULong
+type OSXFsFilCnt is U32
+
+type Gid is U32
+
+type Id is U32
+
+type INo is U64
```

Comment:
It's `ino_t` from the `stat` structure and represents "File serial number."

I didn't know docstrings were possible on type aliases like that. I'll add them.